web-app/client/components/Article.js

```diff
@@ -70,31 +72,36 @@ class Article extends Component {
     document.getElementById('app').style.backgroundColor = backgroundColor;
 
     return (
-      <div id="page-container">
-        {article ? (
-          <div id="single-article" style={{ fontSize, fontFamily, color }}>
-            <Header as="h1" style={{ color }} className="article-title">
-              {title}
-            </Header>
-            Originally from{' '}
-            <a href={sourceUrl}>{publication && publication.name}</a>
-            <p className="article-author">
-              {' '}
-              {author ? `by ${author.name}` : null}
-            </p>
-            <p>
-              {publicationDate
-                ? `Date Published: ${new Date(
-                    publicationDate
-                  ).toLocaleDateString('en-US', dateOptions)}`
-                : null}
-            </p>
-            <div className="article-content">{reactHtmlParser(content)}</div>
-          </div>
-        ) : (
-          <p>Loading... </p>
-        )}
-      </div>
+      <VisContainer>
+        <div id="page-container">
+          {article ? (
+            <div id="single-article" style={{ fontSize, fontFamily, color }}>
+              <Header as="h1" style={{ color }} className="article-title">
+                {title}
+              </Header>
+              Originally from{' '}
+              <a href={sourceUrl}>{publication && publication.name}</a>
+              <p className="article-author">
+                {' '}
+                {author ? `by ${author.name}` : null}
+              </p>
+              <p>
+                {publicationDate
+                  ? `Date Published: ${new Date(
+                      publicationDate
+                    ).toLocaleDateString('en-US', dateOptions)}`
+                  : null}
+              </p>
+              <div className="article-content">{reactHtmlParser(content)}</div>
+              <Segment onClick={history.goBack} className="back-button">
+                <Icon name="long arrow left" Back />back
```

Comment:
Not sure if you caught this in the console: _Warning: React does not recognize the `Back` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `back` instead. If you accidentally passed it from a parent component, remove it from the DOM element._

Reply:
good catch, thanks. fixed.